source/overview/faq.rst

```diff
@@ -435,6 +435,15 @@ The following chart highlights the end user features of Mattermost and their sup
 * X - Full Support
 * O - Partial support
 
+.. _pre-post-proxy:
+
+What are pre-proxy and post-proxy relays?
```

Comment:
This should have a category to be consistent with other major titles,

Reply:
@amyblais It's under `Product Questions`

Reply:
Fixed section header formatting, had used `---` instead of `~~~` by mistake